chat_main.py

```diff
@@ -36,7 +35,6 @@ def ask():
 
     # Set up the language model and predictor
     llm = OctoAiCloudLLM(endpoint_url=endpoint_url)
-    llm_predictor = LLMPredictor(llm=llm)
```

Comment:
This doesn't seem to be necessary for this example and it prevents anyone using python 3.10 from running the example due to [this known issue](https://discuss.pytorch.org/t/dataloader-typeerror-instance-and-class-checks-can-only-be-used-with-runtime-checkable-protocols/139472).

Reply:
Thank you!